lib/yang_wrappers.h

```diff
@@ -172,4 +172,8 @@ extern void yang_dnode_get_ip(struct ipaddr *addr, const struct lyd_node *dnode,
 			      const char *xpath_fmt, ...);
 extern void yang_get_default_ip(struct ipaddr *var, const char *xpath_fmt, ...);
 
+/* mac */
+extern struct yang_data *yang_data_new_mac(const char *xpath,
+					   const struct ethaddr *mac);
+
```

Comment:
Ideally we should try to write the full set of wrapper functions for this new derived type.